analyzers/rspec/cs/S4784_c#.html

```diff
@@ -85,6 +85,4 @@ <h2>See</h2>
   </li>
   <li> OWASP Regular expression Denial of Service - ReDoS </li>
 </ul>
-<h2>Deprecated</h2>
-<p>This rule is deprecated; use {rule:csharpsquid:S2631} instead.</p>
```

Comment:
What's the story behind this? It was undeprecated? I see it deprecated on the RSPEC repo

Reply:
It's strange indeed. The [rule is still deprecated](https://sonarsource.github.io/rspec/#/rspec/S4784/csharp) but the tool remove this anyway.

I will revert the changes.

Reply:
Can you ping Arsenyi? There might be some (maybe already known) glitch in the rule-api

Reply:
Yes, it is a known issue: https://jira.sonarsource.com/browse/RULEAPI-669

Reply:
Maybe with the next release we will follow the logic and mark the rule as ready until we have the replacement

Reply:
Can you prepare PR with rspec update for this rule already? Because we'll forget the context until the next release :/